documentation/manifest.md

```diff
@@ -13,120 +13,122 @@
+    - [x] Remove remote settings from repository
+    - [x] Go to Sync page and publish repository
+    - [x] Go to Team Explorer Connect page (logged in)
+    - [x] Click GitHub > Create
+    - [x] Enter name of original repository
+    - [x] Received error: "There is already a repository named [repo name] for the current account"
 # Connect page when logged in to TFS
-   - [ ] **Connect to a TFS project**
-    - [ ] Login to GitHub
-      - [ ] Team Explorer Connect page: GitHub section appears above TFS section with Clone | Create | Sign out links
-    - [ ] Log out of GitHub
-      - [ ] Team Explorer Connect page: GitHub section appears above TFS section with Clone | Create | Login links
-    - [ ]  Disconnect from TFS (right click on project and "Remove"
+   - [x] **Connect to a TFS project**
+    - [x] Login to GitHub
+      - [x] Team Explorer Connect page: GitHub section appears above TFS section with Clone | Create | Sign out links
+    - [x] Log out of GitHub
+      - [x] Team Explorer Connect page: GitHub section appears above TFS section with Clone | Create | Login links
+    - [ ]  Disconnect from TFS (right click on project and "Remove" (no more option to remove?)
       - [ ] Team Explorer Connect page: GitHub invitation section appears in Hosted Service Providers with Connect.. and Sign up links
 
 # In GitHub pane (logged into GitHub)
-- [ ] **Go to Team Explorer > Home > Pull Requests**
-      - [ ] List of pull requests is displayed.
-      - [ ] Pane title displays "GitHub | (current repository)"
-      - [ ] Selected pane tab title (on the bottom) displays "GitHub"
-      - [ ] Changing the open/close/all filter dropdown filters the pull request list, by selected option (confirm on .com)  
-      - [ ] Changing the Assignee filter dropdown filters the pull request list to display only selected user assigned (confirm on .com)  
-      - [ ] Changing the Author filter dropdown filters the pull request list to display pull requests the user created (confirm on .com)  
-      - [ ] Filter author selection displays recent pull request authors
-      - [ ] "Create New" Link button is displayed
+- [x] **Go to Team Explorer > Home > Pull Requests**
+      - [x] List of pull requests is displayed.
+      - [ ] Pane title displays "GitHub | (current repository)" (title  just says `GitHub`)
+      - [x] Selected pane tab title (on the bottom) displays "GitHub"
+      - [x] Changing the open/close/all filter dropdown filters the pull request list, by selected option (confirm on .com)  
+      - [x] Changing the Assignee filter dropdown filters the pull request list to display only selected user assigned (confirm on .com)  
+      - [x] Changing the Author filter dropdown filters the pull request list to display pull requests the user created (confirm on .com)  
+      - [ ] Filter author selection displays recent pull request authors (recent?  there's a lot on the VisualStudio repo, plenty where list comes up empty)
```

Comment:
Currently it's a alphabetized list of contributors who have made PRs in the past. This step is wrong as well.